package.json

```diff
@@ -51,6 +51,7 @@
     "moment": "^2.24.0",
     "morgan": "^1.9.1",
     "pg": "^7.11.0",
+    "supertest": "^4.0.2",
```

Comment:
We might want to move supertest to dev dependencies

Reply:
My bad, missed a `-D` flag and that slipped through the cracks. Fixing it now....